vm-rust/src/player/movie.rs

```diff
@@ -90,6 +91,7 @@ impl Movie {
       "traceLogFile" => Ok(Datum::String("".to_string())), // TODO
       "traceScript" => Ok(Datum::Int(0)), // TODO
       "movieName" => Ok(Datum::String(self.file_name.to_owned())),
+      "ticks" => Ok(Datum::Int(utils::get_elapsed_ticks(self.timer_tick_start))),
```

Comment:
This prop can be moved to `DirPlayer::get_movie_prop`, that would eliminate the need for a `timer_tick_start` field in the Movie itself.

Reply:
Done.